src/JoomlaBrowser.php

```diff
@@ -911,7 +911,11 @@ public function createMenuItem($menuTitle, $menuCategory, $menuItem, $menu = 'Ma
 		$I->switchToIFrame();
 		$I->debug('I leave time to the iframe to close');
 		$I->wait(2);
-		$I->selectOptionInChosen('Language', $language);
+
+		// @Todo We should use page objects for the selectors
+		$I->scrollTo(['css' => '#jform_language'], 20, 100);
```

Comment:
@yvesh The offset makes sure, that we see the element. I had sometimes the problem, mainly when I scroll up, that the browser scrolls to the begin ob element. But the element was still out of the visible area. In this case I had to use a negative offset, so that browser see the element.